pipelines/preparation/preparation_matched_jointcal_fgcm_tractonly.yaml

```diff
@@ -0,0 +1,9 @@
+description: Produce matched catalogs
+tasks:
+  matchCatalogsTract:
+    class: lsst.faro.preparation.TractMatchedPreparationTask
+    config:
+      python: |
+        config.doApplyExternalPhotoCalib = False
```

Comment:
I expected the following here in order to apply the external calibrations:
```
        config.doApplyExternalPhotoCalib = True
        config.useGlobalExternalPhotoCalib = True
        config.doApplyExternalSkyWcs = True
        config.useGlobalExternalSkyWcs = False
```

Reply:
Yes, should have been (I had changed it for testing, and forgot to revert).